src/parameters/parameters_flow_solver.cpp

```diff
@@ -120,6 +120,55 @@ void FlowSolverParam::declare_parameters(dealii::ParameterHandler &prm)
                               dealii::Patterns::Integer(0, dealii::Patterns::Integer::max_int_value),
                               "Number of mesh refinements for Gaussian bump and naca0012 based cases.");
 
+            prm.declare_entry("use_input_mesh", "false",
+                              dealii::Patterns::Bool(),
+                              "Use the input .msh file which calls read_gmsh. False by default.");
+
+            prm.declare_entry("mesh_reader_verbose_output", "false",
+                              dealii::Patterns::Bool(),
+                              "Flag for verbose (true) or quiet (false) mesh reader output.");
+
+            prm.enter_subsection("boundary_IDs");
+            {
+
+                prm.declare_entry("use_periodic_BC_in_x", "false",
+                                  dealii::Patterns::Bool(),
+                                  "Use periodic boundary condition in the x-direction. False by default.");
+
+                prm.declare_entry("use_periodic_BC_in_y", "false",
+                                  dealii::Patterns::Bool(),
+                                  "Use periodic boundary condition in the y-direction. False by default.");
+
+                prm.declare_entry("use_periodic_BC_in_z", "false",
+                                  dealii::Patterns::Bool(),
+                                  "Use periodic boundary condition in the z-direction. False by default.");
+
+                prm.declare_entry("x_periodic_id_face_1", "2001",
+                                  dealii::Patterns::Integer(1, dealii::Patterns::Integer::max_int_value),
+                                  "Boundary ID for the first periodic boundary face in the x-direction.");
```

Comment:
Its the default value which is arbitrary but since its different for every gmsh file, we need them to be input variables for us to easily use any input mesh file

Reply:
Okay, sounds like they need to be input parameters then, even if it's a little messy.